packages/docs/src/components/landing/Seamless/Seamless.vue

```diff
@@ -89,7 +72,19 @@
 
 <script lang="ts">
 import { Options, Vue } from 'vue-class-component'
-@Options({})
+import SeamlessAnotherButton from './SeamlessAnotherButton.vue'
+import SeamlessAnotherSelect from './SeamlessAnotherSelect.vue'
+import SeamlessAnotherRangeSlider from './SeamlessAnotherRangeSlider.vue'
+import SeamlessAnotherCheckbox from './SeamlessAnotherCheckbox.vue'
+
+@Options({
+  components: {
+    SeamlessAnotherButton,
+    SeamlessAnotherSelect,
+    SeamlessAnotherRangeSlider,
+    SeamlessAnotherCheckbox
+  }
+})
 export default class Seamless extends Vue {
```

Comment:
Can you rename this component to 2 words? SeamlessIntegration maybe?

Reply:
Renamed to SeamlessIntegration